### PR TITLE
Fix Shell panel name in preferences

### PR DESCRIPTION
**Pull request by pskowronek**
Fix Shell panel name in preferences:

obraz

### CODE REVIEW

File: mucommander-translator/src/main/resources/dictionary.properties [hunk 1]
```diff
@@ -861,7 +861,7 @@ prefs_dialog.mail_address = Your email address
 prefs_dialog.mail_server = SMTP server
 prefs_dialog.misc_tab = Misc
 prefs_dialog.confirm_on_quit = Show confirmation dialog on quit
-prefs_dialog.shell = $[RunCommand.label]
+prefs_dialog.shell = Shell
```
Comment (ahadas) [inline mucommander-translator/src/main/resources/dictionary.properties:864]: how about calling it "Terminal" then to reduce the amount of translation items?

Comment (pskowronek) [reply mucommander-translator/src/main/resources/dictionary.properties:864]: Afair, this is for choosing bash vs zsh, so shell is appropriate I think

Comment (pskowronek) [reply mucommander-translator/src/main/resources/dictionary.properties:864]: > how about calling it "Terminal" then to reduce the amount of translation items?

ok, now I'm looking normally thru laptop - yes, you are right. I will change this to Terminal

Comment (pskowronek) [reply mucommander-translator/src/main/resources/dictionary.properties]: @ahadas  PR updated

**Review by ahadas — APPROVED** [review]

**Pull request merged**